app/controllers/kudos_controller.rb

```diff
@@ -71,6 +71,23 @@ def create
     end
   end
 
+  def update
```

Comment:
Are the changes in [this commit](https://github.com/Factual/kudos/pull/17/commits/f5eb43cff32d28a05c6b7649a1b1baa3aa5a8b13) more aligned with what you were thinking? I'm not sure I completely understand strong parameters here.